src/main/scala/io/iohk/ethereum/crypto/package.scala

```diff
@@ -45,6 +46,24 @@ package object crypto {
     generator.generateKeyPair()
   }
 
+  /** @return (privateKey, publicKey) pair */
+  def keyPairToByteArrays(keyPair: AsymmetricCipherKeyPair): (Array[Byte], Array[Byte]) = {
```

Comment:
Well, one returns strings and the other byte arrays. I'd rather say that `AsymmetricCipherKeyPairSerializable.toHexStrings` should use `keyPairToByteArrays` underneath, instead of doing the same things manually.

OTOH, the whole `AsymmetricCipherKeyPairSerializable` seems like a bit of an overkill, especially, that we only need to store private key (we can derive the public key). So, maybe it's best to remove it altogether?